### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by vkatsikaros** [grafana/loki#175]
Hi

I tried to build from source and I got this
```
# git log -1 --oneline 
9b67d6c (HEAD -> master, origin/master, origin/HEAD) Rename instance to tenant to avoid clash with Prom (#164)

# go build ./cmd/loki
# _/root/loki/cmd/loki
cmd/loki/main.go:24:18: cannot use &cfg.Server (type *"github.com/grafana/loki/vendor/github.com/weaveworks/common/server".Config) as type *"github.com/cortexproject/cortex/vendor/github.com/weaveworks/common/server".Config in argument to "github.com/cortexproject/cortex/pkg/util".InitLogger

# go build ./cmd/promtail
# _/root/loki/cmd/promtail
cmd/promtail/main.go:25:18: cannot use &config.ServerConfig (type *"github.com/grafana/loki/vendor/github.com/weaveworks/common/server".Config) as type *"github.com/cortexproject/cortex/vendor/github.com/weaveworks/common/server".Config in argument to util.InitLogger
```

I am new to go lang, so I apologize if the question is stupid (as I see the cicrleCI is green right now)

My env (fresh ubuntu 18.04 installation)
```
# go env
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/root/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build735073857=/tmp/go-build -gno-record-gcc-switches"

# go version
go version go1.11.4 linux/amd64
```

**Comment by gouthamve** [comment]
@vkatsikaros Fixed in #176

**Comment by daixiang0** [comment]
I use go image to test:
```
root@b56ef4650f52:/go# go get github.com/grafana/loki
can't load package: package github.com/grafana/loki: no Go files in /go/src/github.com/grafana/loki
root@b56ef4650f52:/go# go version
go version go1.11.3 linux/amd64
```
